README.md

```diff
@@ -1,14 +1,21 @@
-# vehicle-commander-qt
+# message-simulator-qt
 
-The Vehicle Commander Qt demonstrates developer samples for building and testing in-vehicle military applications with the ArcGIS Runtime Qt SDK.  The Vehicle Commander Qt projects contains source code for creating sample in-vehicle, touchscreen, and support applications.
+Message Simulator provides UDP messaging for applications and system components in the ArcGIS for the Military solution, including:
 
-![Image of Vehicle Commander]( ScreenShot.jpg "vehicle-commander-qt")
+* [Vehicle Commander](https://github.com/Esri/vehicle-commander-java)
+* [Squad Leader](https://github.com/Esri/squad-leader-android)
+* Custom transports and adapters for ArcGIS GeoEvent Extension for Server ([solutions-geoevent-java](https://github.com/Esri/solutions-geoevent-java))
```

Comment:
Linking to both, as well as source and binaries for both Squad Leader and Vehicle Commander. Done in b7f0490e6fee4f267a8285a8cd190089ae43a921.